app/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
+    implementation 'junit:junit:4.13.2'
```

Comment:
Why would this be needed when it's used in tests only? `testImplementation` should be fine.

Reply:
Idk. Still don't understand everything in gradle. So what's the difference if it's testImplementation?

Reply:
`testImplementation` is used to specify what gets imported into test sources - under test dir only; not available in main, and will not get built into the app (apk/aab).
 - `implementation` is for `main`
 - `testImplementation` is for `test`
 -  `androidTestImplementation` is for `androidTest` 
These are the default source dirs. There is a way to specify which dirs to include for which sources though explicitly if needed - can be more than one.
```Kotlin
sourceSets {
    main {
        java {
             srcDirs = ['src', 'src2']
        }
        resources {
            srcDir 'src/resources'
        }
    }
    test {
        java {
            srcDirs = ['test', 'test2']
        }
    }
    integrationTest {
        java {
            srcDirs = ['androidTest']
        }
    }
 }
```
<img width="168" alt="image" src="https://github.com/FilipM79/Foody/assets/18649632/b43e100e-838e-403c-8a9a-00690572c068">